token/cli/src/config.rs

```diff
@@ -22,6 +22,8 @@ pub(crate) enum KeypairOrPath {
 }
 
 pub(crate) struct Config<'a> {
+    pub(crate) default_signer: Arc<dyn Signer>,
+    pub(crate) default_address: Pubkey,
     pub(crate) rpc_client: Arc<RpcClient>,
     pub(crate) websocket_url: String,
     pub(crate) output_format: OutputFormat,
```

Comment:
I can't comment further down, but with this change, it doesn't seem like we even need `default_keypair` anymore! Let's remove it